### DISCUSSION TITLE
WLAN-Pi Availability?

### DISCUSSION THREAD

**Discussion by netvic** [WLAN-Pi/feedback#30] [General Inquiries] [answered]
I just learned about this project and was hoping to order one already built but can't seem to find any available.  The wlanprofessionals link shows a blank page and the badgerwifi store doesn't seem to have any in stock.

I'm mainly interested in the Pro model, but haven't found any models available yet.

Any suggestions?

**Comment by jolla** [comment, marked as answer]
Thanks for your interest in the project. Due to the supply chain issues it is unknown when the WLAN Pi Pro models will be available again. Two new WLAN Pi models have been announced (WLAN Pi M4 and WLAN Pi R4). You can find out more details at the following link. https://userguide.wlanpi.com/hardware/compare-wlan-pi-models 

The R4 uses a Raspberry Pi 4 at its core, if you have one laying around or are able to find one in stock that would be the best way to get started.

**Reply by nickjvturner** [reply]
Kits and complete units available on https://badgerwifi.co.uk
Or DIY:
parts list: https://userguide.wlanpi.com/hardware/wlan-pi-r4
3D model files: https://www.printables.com/model/265668-wlan-pi-r4-fascia-essentials
